nav2_motion_primitives/include/nav2_motion_primitives/back_up.hpp

```diff
@@ -19,20 +19,21 @@
 #include <memory>
 
 #include "nav2_motion_primitives/motion_primitive.hpp"
-#include "nav2_tasks/back_up_task.hpp"
+#include "nav2_msgs/action/back_up.hpp"
 
 namespace nav2_motion_primitives
 {
+using BackUpAction = nav2_msgs::action::BackUp;
 
-class BackUp : public MotionPrimitive<nav2_tasks::BackUpCommand, nav2_tasks::BackUpResult>
+class BackUp : public MotionPrimitive<BackUpAction>
```

Comment:
If we're changing to actions, can we delete the `XYZTask` files along with it?

Reply:
Yes, I'm pushing this change to the "Lifecycle" branch on which these have been deleted already.